test/ConfigTraitTest.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-authentication-oauth2 for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (https://www.zend.com)
```

Comment:
Here and elsewhere you introduce new test classes: since the file was introduced this year, use 2018 as the copyright year.